metadata/elasticsearch/src/main/java/com/spotify/heroic/metadata/elasticsearch/MetadataBackendKV.java

```diff
@@ -592,6 +610,20 @@ private static void buildContext(final XContentBuilder b, Series series) throws
         }
         b.endArray();
 
+        if (indexResourceIdentifiers) {
+          b.startArray(RESOURCE);
+          for (final Map.Entry<String, String> entry : series.getResource().entrySet()) {
+            b.value(entry.getKey() + TAG_DELIMITER + entry.getValue());
+          }
+          b.endArray();
+
+          b.startArray(RESOURCE_KEYS);
```

Comment:
Do we need this? Can we maybe verify with unit tests? If it's not used we could drop it as it will be additional size in ES.

Reply:
I this in reference to the `RESOURCE_KEYS`?

Reply:
yes, sorry I wasn't clear